Data/Services/CustomerService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Data.Entities;
+
+namespace Data.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private List<Customer> _customerList = new List<Customer>();
+
+        public virtual void Create(Customer customer)
```

Comment:
Por que os métodos estão como Virtual? métodos virtuais podem ser sobrescrevidos por quem for utilizar a classe e isso não será necessário